src/main/java/com/julienvey/trello/domain/TrelloEntity.java

```diff
@@ -10,7 +10,16 @@
     @JsonIgnore
     protected Trello trelloService;
 
-    public void setInternalTrello(Trello trelloService) {
+    @SuppressWarnings("unchecked")
+    public <T extends TrelloEntity> T setInternalTrello(Trello trelloService) {
```

Comment:
Gives users ability fluently chain methods. This change is backward compatible.

Reply:
to be fair, this will force people to change their code if they extended TrelloEntitry

Reply:
Sure, I've miss that, but without that we can't provide a solid fluent API.